Tests/IGListStackSectionControllerTests.m

```diff
@@ -85,8 +85,9 @@ - (void)test_whenInitializingStack_thatNumberOfItemsMatches {
     section4.items = 1;
 
     IGListStackedSectionController *stack = [[IGListStackedSectionController alloc] initWithSectionControllers:@[section1, section2, section3, section4]];
-
-    XCTAssertEqual([stack numberOfItems], 6);
+    [stack performBatchAnimated:false updates:^{} completion:^(BOOL finished) {
+        XCTAssertEqual([stack numberOfItems], 6);
```

Comment:
@rnystrom - let me know what you think about just using:
`    [stack performSelectorOnMainThread:@selector(reloadData) withObject:nil waitUntilDone:TRUE];
`
to get the unit tests working.